utils/validators/container-images.js

```diff
@@ -0,0 +1,18 @@
+export function containerImages(spec, getters, errors) {
+  let container;
+
+  if (spec.jobTemplate) {
+    // cronjob pod template is nested slightly different than other types
+    const { jobTemplate: { spec: { template: { spec: { containers } } } } } = spec;
+
+    container = containers[0];
```

Comment:
I don't think that can happen because of the defaults applied in the UI; containers should always be at least an empty array. I added some precautionary defaults here anyway.